src/main/java/seedu/command/attendance/AddAttendanceList.java

```diff
@@ -0,0 +1,102 @@
+package seedu.command.attendance;
+
+import seedu.attendance.Attendance;
+import seedu.attendance.AttendanceList;
+import seedu.command.Command;
+import seedu.exception.DukeException;
+import seedu.parser.AttendanceParser;
+import seedu.ui.DisplayList;
+import seedu.ui.UI;
+
+import java.util.ArrayList;
+
+import static seedu.duke.Duke.studentListCollection;
+
+public class AddAttendanceList extends Command {
+
+    static UI ui = new UI();
+    AttendanceList attendances;
+    boolean isByNameList;
+    String eventName;
+    DisplayList displayList;
```

Comment:
package-private? For Unit Test?

Reply:
Also why need declare ui as static?

Reply:
my mistake, ui should be static. I will also declare all the types